### PR TITLE
Adds user agent to query headers

### PR DESCRIPTION
**Pull request by gschorkopf**
Issues:

* I don't know how to test this. From a python console:

```python
>>> from fastly._version import __version__
>>> __version__
'0.1.1'
```

Which seems to work OK in that context.

Risk:

* Potentially a lot because of ^

### CODE REVIEW

**Comment by thommahoney** [comment]
This looks good tome.

**Comment by blithe** [comment]
I'd love to see a test that asserts the value of the header.

File: fastly/connection.py [hunk 1]
```diff
@@ -19,6 +20,8 @@ def __init__(self, host='api.fastly.com', secure=True, port=None, root='',
         self.http_conn = None
     
     def request(self, method, path, body=None, headers={}):
+        headers['User-Agent'] = 'fastly-py-v{}'.format(__version__)
+
```
Comment (gschorkopf) [inline fastly/connection.py]: I added a `print headers` here, and all tests included our `'User-Agent'` key/value pair.

File: test/api_test.py [hunk 2]
```diff
@@ -57,5 +57,9 @@ def test_auth_session_success(self):
         self.api.authenticate_by_password(self.user, self.password)
         self.api.conn.request('GET', '/current_customer')
 
+    def test_sets_default_user_agent_header(self):
```
Comment (gschorkopf) [inline test/api_test.py]: This doesn't necessarily test `request`, just `Connection __init__`

Comment (gschorkopf) [inline test/api_test.py]: Thoughts?

**Comment by drbrain** [comment]
We should increment the version to `0.2.0` because the User-Agent header is a new feature.

**Comment by gschorkopf** [comment]
I was under the impression we do that after the PR is accepted @drbrain?

**Comment by blithe** [comment]
Merging this as is and we can bump the version on master.

**Pull request merged**